web/api/v0/__init__.py

```diff
@@ -252,9 +271,13 @@ def post(self) -> ResponseReturnValue:
 
 
 class UserByIPResource(Resource):
-    def get(self) -> ResponseReturnValue:
-        ipv4 = request.args.get('ip', IPAddress)
-
+    @use_kwargs(
+        {
+            "ipv4": fields.IP(required=True, data_key="ip"),
```

Comment:
Not sure what to do about the type warning here; mypy considers the `IP.__init__` function untyped due to lack of `-> None`. nothing to be done about it. I don't want to suppress this error on every invocation tho…
Plus, stubs (`.pyi`) cannot augment existing lib type hints, they act as replacements.